Cargo.lock

```diff
@@ -0,0 +1,800 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Does this file need to be in the repo or can it just be generated by the user once using the cargo config files?

Reply:
uhh we include our yarn.lock in finishline but idk honestly